packages/server/src/index.ts

```diff
@@ -99,8 +100,39 @@ async function onInitialized() {
 }
 
 function loadTs() {
+	const tsserverPath = findTsserverPath()
+	connection.console.info(`Loaded tsserver: '${tsserverPath}'`);
 	return {
-		server: shared.loadTypescript(options.typescript.serverPath),
-		localized: options.typescript.localizedPath ? shared.loadTypescriptLocalized(options.typescript.localizedPath) : undefined,
+		server: shared.loadTypescript(findTsserverPath()),
+		localized: options.typescript?.localizedPath ? shared.loadTypescriptLocalized(options.typescript.localizedPath) : undefined,
+	}
+}
+
+function findTsserverPath(): string {
+	if (options.typescript?.serverPath) {
+		return options.typescript.serverPath;
 	}
+
+	const tsServerPath = path.join('typescript', 'lib', 'tsserver.js');
+
+	// look into node_modules of workspace root
+	let executable = findPathToModule(folders[0], tsServerPath);
```

Comment:
I used the first workspace folder `folders[0]` and I know that this is not right.
But I do not know how to handle multiple workspace folders... 

Any suggestions are welcome. :)